pages/components/Navigation.jsx

```diff
@@ -0,0 +1,59 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export const Navigation = () => {
+  return (
+    <StyledNav>
+      <li>
+        <Link href="/">
+          <a>
+            {" "}
+            <img src="/Icon/Home.svg" alt="home" width="50px" height="50px" />
+          </a>
+        </Link>
+      </li>
+      <li>
+        <Link href={"/addCard"}>
+          <a>
+            <img src="/Icon/Plus.svg" alt="add" width="50px" height="50px" />
+          </a>
+        </Link>
+      </li>
+      <li>
+        <Link href="/enterVote">
+          <a>
+            <img src="/Icon/vote.png" alt="vote" width="50px" height="50px" />
+          </a>
+        </Link>
+      </li>
+      <li>
+        <Link href="/showResults">
+          <a>
+            <img
+              src="/Icon/Results.svg"
```

Comment:
Your results icon doesn't load, cause it is lowercase in your Icon folder:
```suggestion
              src="/Icon/results.svg"
```